pages/star-search/waitlist.tsx

```diff
@@ -148,28 +184,43 @@ export default function StarSearchWaitListPage({
         twitterCard="summary_large_image"
       />
       <ProfileLayout>
-        <div className="flex flex-col items-center gap-4 px-2 mb-8 sm:pt-8 md:pt-0 lg:w-99">
+        <div className="flex flex-col items-center gap-4 px-2 mb-8 sm:pt-8 md:pt-0 lg:w-99" aria-live="polite">
           <div className="flex gap-2 items-center">
             <Image src="/assets/star-search-logo.svg" alt="" width={40} height={40} />
             <h1 className="text-3xl lg:text-4xl font-bold bg-clip-text text-transparent bg-gradient-to-r from-sauced-orange to-amber-400">
               StarSearch
             </h1>
           </div>
-          <p className="font-semibold text-5xl text-center tracking-tight text-balance">Copilot, but for git history</p>
-          <p className="text-center">Ask anything, get AI powered insights based on contributor data</p>
+          <p className="font-semibold text-5xl text-center tracking-tight text-balance">
+            {getTitle({ sessionToken, isWaitlisted, waitlistCount, alreadyOnWaitlist })}
+          </p>
+          <p className="text-center">
+            {isWaitlisted
+              ? "Share with others to get early access sooner"
+              : "Ask anything, get AI powered insights based on contributor data"}
```

Comment:
nit: more of a question with naming?

```suggestion
              : "Ask anything, get AI-powered insights based on contributor data"}
```